src/Microsoft.ServiceFabric.Actors/Runtime/ActorTypeInformation.cs

```diff
@@ -125,10 +125,10 @@ public static ActorTypeInformation Get(Type actorType)
                 actorServiceName = actorServiceAttr.Name;
             }
 
-            // get all actor interfaces
+            // Gets all actor interfaces.
             var actorInterfaces = actorType.GetActorInterfaces();
 
-            // ensure that the if the actor type is not abstract it implements at least one actor interface
+            // Ensures if the actor type is not abstract, it implements at least one actor interface.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.